ftw/noticeboard/browser/conditions.py

```diff
@@ -0,0 +1,17 @@
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope.publisher.browser import BrowserView
+from ftw.noticeboard.content.category import INoticeCategory
+from ftw.noticeboard.content.notice import INotice
+
+
+class TermsAndConditions(BrowserView):
+
+    template = ViewPageTemplateFile('templates/conditions.pt')
+
+    def __call__(self):
+        self.category = None
+        if INoticeCategory.providedBy(self.context):
+            self.category = self.context
+        elif INotice.providedBy(self.context):
+            self.category = self.context.aq_parent
```

Comment:
Would it be possible to add `INotice` to a content not providing `INoticeCategory`?

Reply:
Yes that's possible, but you have to configure it that way on purpose. The default config does not allow adding Notices anywhere else than in Categories.